src/marqo/core/search/hybrid_search.py

```diff
@@ -83,24 +83,36 @@ def search(
 
         marqo_index = index_meta_cache.get_index(config=config, index_name=index_name)
 
-        # TODO: Remove when we support unstructured.
-        if isinstance(marqo_index, UnstructuredMarqoIndex):
-            raise core_exceptions.UnsupportedFeatureError(
-                "Unstructured indexes are not yet supported for hybrid search. "
-                "Please use a structured index.")
-
+        # Version checks (different for structured and unstructured)
         marqo_index_version = marqo_index.parsed_marqo_version()
-        if marqo_index_version < constants.MARQO_HYBRID_SEARCH_MINIMUM_VERSION:
+        if isinstance(marqo_index, StructuredMarqoIndex) and \
+                marqo_index_version < constants.MARQO_STRUCTURED_HYBRID_SEARCH_MINIMUM_VERSION:
+            raise core_exceptions.UnsupportedFeatureError(
+                f"Hybrid search is only supported for Marqo structured indexes created with Marqo "
+                f"{constants.MARQO_STRUCTURED_HYBRID_SEARCH_MINIMUM_VERSION} or later. "
+                f"This index was created with Marqo {marqo_index_version}."
+            )
+        elif isinstance(marqo_index, UnstructuredMarqoIndex) and \
+                marqo_index_version < constants.MARQO_UNSTRUCTURED_HYBRID_SEARCH_MINIMUM_VERSION:
             raise core_exceptions.UnsupportedFeatureError(
-                f"Hybrid search is only supported for Marqo indexes created with Marqo "
-                f"{constants.MARQO_HYBRID_SEARCH_MINIMUM_VERSION} or later. This index was created with Marqo "
-                f"{marqo_index_version}."
+                f"Hybrid search is only supported for Marqo unstructured indexes created with Marqo "
+                f"{constants.MARQO_UNSTRUCTURED_HYBRID_SEARCH_MINIMUM_VERSION} or later. "
+                f"This index was created with Marqo {marqo_index_version}."
             )
 
         # Use default hybrid settings if not provided
         if hybrid_parameters is None:
             hybrid_parameters = HybridParameters()
 
+        # TODO: Remove when unstructured searchable attributes are supported
+        if isinstance(marqo_index, UnstructuredMarqoIndex) and \
+                (hybrid_parameters.searchableAttributesTensor is not None or \
```

Comment:
Do we not support specifying attributes to rank by?

Reply:
We don't yet, it's still a fast follower.

Reply:
Also, this whole functionality is blocked off until we support searchable attributes with unstructured (2.12)